app/SectionApp/TodoContainer/Tasks/CreateStepsTask.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\SectionApp\TodoContainer\Tasks;
+
+use App\SectionApp\TodoContainer\Repositories\Interfaces\StepRepositoryInterface;
+use Spatie\LaravelData\DataCollection;
+use App\SectionApp\TodoContainer\Values\CreateOrUpdateStepValue;
+
+/**
+ * Class CreateStepsTask
+ *
+ * This class is responsible for creating a new step
+ */
+class CreateStepsTask
+{
+    /**
+     * Class constructor
+     */
+    public function __construct(protected StepRepositoryInterface $stepRepository)
+    {
+    }
+
+    /**
+     * Handle the task.
+     *
+     * @param DataCollection<int, CreateOrUpdateStepValue> $steps
+     */
+    public function handle(DataCollection $steps, int $idTodo): void
+    {
+        $data = [];
+        foreach ($steps as $step) {
+            $item = [
+                "todo_id" => $idTodo,
+                "description" => $step->description,
+                'order' => $step->order
+            ];
+            $data[] = $item;
+        }
```

Comment:
Có thể viết gọn hơn.
code chưa test
```suggestion
        $data = $steps->toArray();
        data_fill($data, '*.todo_id', $todoId);
```